src/components/Search/SearchInput.tsx

```diff
@@ -197,8 +192,18 @@ const SearchInput = ({ onStateChange }: SearchInputListener) => {
       className={isExpanded ? 'pf-u-flex-grow-1' : 'chr-c-search__collapsed'}
     />
   );
+
+  let menuFooter;
+  if (searchItems.length > 0 && !isFetching) {
+    menuFooter = (
+      <MenuFooter className="pf-v5-u-px-md">
+        <SearchFeedback query={searchValue} results={searchItems} />
+      </MenuFooter>
+    );
+  }
+
   const menu = (
-    <Menu ref={menuRef} className="pf-v5-u-pt-sm pf-v5-u-px-md chr-c-search__menu">
+    <Menu ref={menuRef} className="pf-v5-u-pt-sm chr-c-search__menu">
```

Comment:
This width utility class moved to the menu group below (and is also present on the menu footer) to get us closer to the end design in our mockups. Some additional styling will be required later.